conda_build/index.py

```diff
@@ -6,7 +6,7 @@
 from os.path import dirname
 
 from conda.base.context import context
-from conda.core.index import get_index
+from conda.core.index import Index
```

Comment:
In principle, we won't need this during the deprecation period, right? So we can release a new conda without this PR and it _should_ work, correct?

Reply:
Yes.